src/Strategy.sol

```diff
@@ -294,9 +311,21 @@ contract Strategy is BaseStrategy {
     // transfers all tokens to new strategy
     function prepareMigration(address _newStrategy) internal override {
         // want is transferred by the base contract's migrate function
-        sanTokenGauge.withdraw(balanceOfStakedSanToken());
+        // sanTokenGauge.withdraw(balanceOfStakedSanToken());
+
+        uint256 _angleBalance = balanceOfAngleToken();
+        if(_angleBalance > 0) {
```

Comment:
Extreme nit :) : can put a space between 'if' and '(' to make it consistent with the below

Reply:
hah done - good eye